pkg/driver/helmdriver.go

```diff
@@ -70,12 +70,13 @@ func (d *helmDriver) Initialize(ctx context.Context, clusterName string) (err er
 }
 
 func (d *helmDriver) Install(ctx context.Context,
-	name string, namespace string, source api.PackageOCISource, values map[string]interface{}) error {
+	name string, namespace string, createNamespace bool, source api.PackageOCISource, values map[string]interface{}) error {
 	var err error
 	install := action.NewInstall(d.cfg)
 	install.Version = source.Version
 	install.ReleaseName = name
 	install.Namespace = namespace
+	install.CreateNamespace = createNamespace
```

Comment:
Does helm fail installs if the namespace already exists?

Reply:
No, I tested that both on a workload and management target cluster.